### DISCUSSION TITLE
Edit video

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#2295] [Ideas] [answered]
### Is your idea related to a problem?

It's not a problem, but it will help a lot.

### Describe your idea

The M3 already has a similar feature, however, the need has arisen to use an excerpt from a JW Stream video. The **jw sign language** has a _similar_ feature, the idea is to take a section of the video and edit it to the point we want to present without appearing in the broadcast, that is, it only appears after play and saves the change in case of restarting the PC.

![image](https://github.com/sircharlo/meeting-media-manager/assets/90984283/114be4ce-f962-4fc1-ba7f-eebf6088070e)

### Describe alternatives you've considered

Currently in m3, you have the option of setting the time you want the video to start playing, however, if you close m3, the schedule will not be saved. And the idea would be to use some excerpts from the recorded meeting as approved for use.

### Additional context

I even tried to use the available resource of putting the start and end time in the video, in this week's schedule, it doesn't work in the last part of the video (Bible Study)

Bible Reading OK
Starts 26:07
Ends 28:32
![image](https://github.com/sircharlo/meeting-media-manager/assets/90984283/3760eff0-6949-4f6e-b241-9ed5a6f4d728)

First Conversation OK
Starts: 30:07
Ends: 31:48

Revisit OK
Starts: 33:21
Ends: 36:29

Bible Study [Time doesn't work in m3]
Starts: 37:39
Ends: 42:06
![image](https://github.com/sircharlo/meeting-media-manager/assets/90984283/34bcca16-b7ec-4da9-bbeb-7269f09f8539)

Another detail is that this playback time setting is not saved if you close M3 or do it via the server. As seen in the example above, the congregation has been approved to use JW Stream for a while, so we have a need to use this feature frequently. Can you help us @mtdvlpr ?

**Comment by mtdvlpr** [comment]
Hi @e-Feitosa, I'll see what I can do. For the error, it would help if I have the video file, so I can reproduce it. For now, I would suggest using a simple video cutting program to cut the full JW stream into the different parts you need.

**Reply by e-Feitosa** [reply]
I sent it by email

**Comment by mtdvlpr** [comment, marked as answer]
@e-Feitosa, the bug should be fixed in [the latest beta release](https://github.com/sircharlo/meeting-media-manager/releases/tag/v23.11.1-beta.3). Could you test this?

**Reply by e-Feitosa** [reply]
Excellent, the BUG has **been resolved** regarding the incorrect time. Thank you very much.

**Comment by alanfermtz** [comment]
How can you enter to timestamp mode?

**Comment by e-Feitosa** [comment]
You can double click the video time to edit 
![image](https://github.com/sircharlo/meeting-media-manager/assets/90984283/2bd07e2c-a9d0-4fc2-904b-5d69ab1ad05c)
